test/list-indentation-first-line-indented.md

```diff
@@ -0,0 +1,9 @@
+# Test
+
+   * item 1
+   * item 2
+      * item 2.1
+      * item 2.2
+         * item 2.2.1
+      * item 2.3
+   * item 3
```

Comment:
Please also add an example list without indent here to verify it’s reported as a violation. And make sure some other file tests for indent being used without the parameter as a violation - or add one.